gwakeonlan/settings.py

```diff
@@ -43,6 +44,12 @@ def __init__(self):
     parser.add_option('-q', '--quiet', dest='verbose_level',
                       action='store_const', const=VERBOSE_LEVEL_QUIET,
                       help='hide error and information messages')
+    parser.add_option('-i', '--import-ethers-file', dest='ethers_file',
+                      help='Include the given ethers file')
```

Comment:
ARP cache import is somewhat interative, but the ethers file import is not. Thats why an CLI interface makes sense for it. As you said, it does not make any difference for the GUI but it can be a nice feature for power users.